src/crafting_gui.cpp

```diff
@@ -513,7 +513,12 @@ const recipe *select_crafting_recipe( int &batch_size )
                 if( last_recipe != current[line] ) {
                     last_recipe = current[line];
                     tmp = current[line]->create_result();
-                    tmp.info( true, thisItem );
+                }
+                if( batch ) {
```

Comment:
Both branches contain the very same code, why have a branch here at all?

Reply:
My bad, this was a left over from testing. Fixed.